sk-csharp-chatgpt-plugin/azure-function/config-samples/appsettings.json.azure-example

```diff
@@ -0,0 +1,25 @@
+{
+  "kernel": {
+    "serviceType": "AzureOpenAI",
+    "serviceId": "text-davinci-003",
+    "deploymentOrModelId": "text-davinci-003",
+    "endpoint": "https:// ... your endpoint ... .openai.azure.com/"
+  },
+  "aiPlugin": {
+    "schemaVersion": "v1",
+    "nameForModel": "MyPlugin",
+    "nameForHuman": "My Plugin",
+    "descriptionForModel": "<Instructions for the model>",
+    "descriptionForHuman": "<Description for users>",
```

Comment:
Can these two values be filled out with the correct entries for the Joke prompt?